src/main/java/org/apache/commons/compress/archivers/zip/ZipArchiveOutputStream.java

```diff
@@ -365,8 +365,33 @@ public ZipArchiveOutputStream(final Path file, final OpenOption... options) thro
      * @since 1.20
      */
     public ZipArchiveOutputStream(final File file, final long zipSplitSize) throws IOException {
+        this(file.toPath(), zipSplitSize);
+    }
+
+    /**
+     * Creates a split ZIP Archive.
+     *
+     * <p>The files making up the archive will use Z01, Z02,
+     * ... extensions and the last part of it will be the given {@code
+     * file}.</p>
+     *
+     * <p>Even though the stream writes to a file this stream will
+     * behave as if no random access was possible. This means the
+     * sizes of stored entries need to be known before the actual
+     * entry data is written.</p>
+     *
+     * @param path the file that will become the last part of the split archive
+     * @param zipSplitSize maximum size of a single part of the split
+     * archive created by this stream. Must be between 64kB and about
+     * 4GB.
+     *
```

Comment:
No need for extra whitespace.

Reply:
This is a copy paste from the existing javadocs of methods from within the class, if you want I can correct them all for white-spaces.

Reply:
I prefer a focused PR, new lines should be "correct". If you want to update unrelated lines, feel free to do so in a _different_ PR, please. This makes reviewing simpler and the PR itself only about one item.

Reply:
Tried my best to undo unrelated changes and keep new things compact and correct.